### PR TITLE
Always ignore stdin for docker-cred list checks.

### PR DESCRIPTION
**Pull request by ericpromislow**
Signed-off-by: Eric Promislow <epromislow@suse.com>

### CODE REVIEW

**Review by mook-as — APPROVED** [review]
Context: we're hitting issues on Linux, where we sometimes see `EPIPE` when the input isn't read.

This will hopefully fix #2329

File: src/utils/dockerDirManager.ts [hunk 1]
```diff
@@ -239,11 +236,7 @@ export default class DockerDirManager {
     } else if (platform === 'darwin') {
       return 'osxkeychain';
     } else if (platform === 'linux') {
-      if (currentCredsStore === 'secretservice') {
-        return 'secretservice';
-      } else {
-        return 'pass';
-      }
+      return 'secretservice';
```
Comment (jandubois) [inline src/utils/dockerDirManager.ts:239]: This has nothing to do with ignoring stdin and seems like a change in functionality.

**Review by jandubois — CHANGES_REQUESTED** [review]
Changing the default credstore on Linux from `pass` to `secretservice` has not been discussed/agreed on.

**Review by mook-as — DISMISSED** [review]
Seems to fix the `EPIPE` issue.

Tested on OpenSUSE Leap 15.3 (installed `password-store` and `gnutls`).

**Review by mook-as — APPROVED** [review]
The PR doesn't really do what it says on the tin anymore, but :shrug:

**Review by mook-as — APPROVED** [review]

**Pull request merged**